optimizely/decision_service.py

```diff
@@ -489,3 +489,63 @@ def get_variation_for_feature(self, project_config, feature, user_context, optio
         if rollout_variation_reasons:
             decide_reasons += rollout_variation_reasons
         return variation, decide_reasons
+
+    def validated_forced_decision(self, decision_context, user_context):
+        """
+        Gets forced decisions based on flag key, rule key and variation.
+
+        Args:
+            decision context: a decision context
+            user_context context: a user context
+
+        Returns:
+            Variation of the forced decision.
+        """
+        reasons = []
+
+        forced_decision = user_context.find_forced_decision(decision_context)
```

Comment:
Since we remove extra checking from `get_forced_decision`,  we do not need `find_forced_decision`. It's identical to `get_forced_decision`. We can remove it and use `get_forced_decision` here.